test/samples/simple/index.js

```diff
@@ -0,0 +1,17 @@
+import { withComponent, withRenderer } from '../../../src';
+
+let Component = withComponent(withRenderer());
```

Comment:
any particular reason why are you using withRenderer ? it's already within withComponent

Reply:
None; I'll remove it.